src/avatar.jsx

```diff
@@ -156,7 +154,7 @@ const Avatar = React.createClass({
           className={className}
         >
           {icon && React.cloneElement(icon, {
-            color: color,
+            color: color || avatar.color,
             style: Object.assign(styles.icon, icon.props.style),
```

Comment:
I guess here I mean, any reason as a general rule we wouldn't just document if you want to change the icons style, just pass it in a style prop?

And if we need to provide a default, we would just add it in the assign chain.